NextcloudTalk/NCChatViewController.m

```diff
@@ -3840,9 +3847,24 @@ - (void)cellWantsToDownloadFile:(NCMessageFileParameter *)fileParameter
 
 - (void)cellHasDownloadedImagePreviewWithHeight:(CGFloat)height forMessage:(NCChatMessage *)message
 {
+    BOOL isAtBottom = [self shouldScrollOnNewMessages];
```

Comment:
```suggestion
    if (message.file.previewImageHeight == height) {
        return;
    }

    BOOL isAtBottom = [self shouldScrollOnNewMessages];
```
Prevent reloading the cells, if we already know the height. If we don't use `ceil` (see above), this check might fail.